ONSdigital/ras-frontstage
# Upgrade python version to 3.12

# What and why?
Upgrades python to 3.12 as part of ongoing tech upgrades. 

# How to test?
Remove your existing pipenv with 3.11
Install python 3.12 (I used pyenv) either globally or a new local version.
Create a new environment with pipenv install 
Check your python interpreter has updated or add a new one based off the created pipenv if it isn't using 3.12
Run the tests and app locally 
Deploy this PR, run acceptance tests and check frontstage is working correctly. 

# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-1379
